spacetemplate/assets/legacy.yaml

```diff
@@ -87,6 +87,42 @@ work_item_types:
   - *experienceID
   - *valuePropositionID
 
+board_config:
+
+- id: "24181b5c-713f-4bef-a19f-45240875da92"
+  name: Scenarios Board
+  description: This is the default board config for the legacy template (Scenarios).
+  context: "679a563c-ac9b-4478-9f3e-4187f708dd30"
+  contextType: "TypeLevelContext"
+  columns:
+  - id: "b4edad70-1d77-4e5a-b973-0f0d599fd20d"
+    title: "New"
+    order: 0
+    transRuleKey: "updateStateFromColumnMove"
+    transRuleArguments: "{ metaState: 'mNew' }"
+  - id: "b4edad70-1d77-4e5a-b973-0f0d599fd20d"
+    title: "In Progress"
+    order: 0
+    transRuleKey: "updateStateFromColumnMove"
+    transRuleArguments: "{ metaState: 'mInprogress' }"
+
+- id: "56d62801-798a-4bb0-9c97-89f136f3d539"
+  name: Experiences Board
+  description: This is the default board config for the legacy template (Experiences).
+  context: "8e4c995d-8e5f-4ad6-85bd-3cfbe718f908"
+  contextType: "TypeLevelContext"
+  columns:
```

Comment:
Instead of defining the `columns` again for each board, you could define the columns once and refer them wherever required.

```
board_columns:
  - id: &New "b4edad70-1d77-4e5a-b973-0f0d599fd20d"
    title: "New"
    order: 0
    transRuleKey: "updateStateFromColumnMove"
    transRuleArguments: "{ metaState: 'mNew' }"
  - id: &InProgress "b4edad70-1d77-4e5a-b973-0f0d599fd20d"
    title: "In Progress"
    order: 0
    transRuleKey: "updateStateFromColumnMove"
    transRuleArguments: "{ metaState: 'mInprogress' }"


board_config:

- id: "24181b5c-713f-4bef-a19f-45240875da92"
  name: Scenarios Board
  description: This is the default board config for the legacy template (Scenarios).
  context: "679a563c-ac9b-4478-9f3e-4187f708dd30"
  contextType: "TypeLevelContext"
  columns:
          - *New
          - *InProgress
                

- id: "56d62801-798a-4bb0-9c97-89f136f3d539"
  name: Experiences Board
  description: This is the default board config for the legacy template (Experiences).
  context: "8e4c995d-8e5f-4ad6-85bd-3cfbe718f908"
  contextType: "TypeLevelContext"
  columns:
          - *New
          - *InProgress
```

Reply:
All columns are unique! The InProgress column for board0 is not the same as for board1, so I don't think this makes sense. That they are named equally is just a coincidence. They are completely different instances and work items will refer to them independently.